project_templates/test_report/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}/{{cookiecutter.series.upper()}}-{{cookiecutter.serial_number}}plan.tex

```diff
@@ -0,0 +1,44 @@
+\documentclass[DM,lsstdraft,toc]{lsstdoc}
+\usepackage{geometry}
+\usepackage{longtable,booktabs}
+\usepackage{enumitem}
+\usepackage{arydshln}
+
+\input{meta.tex}
+
+\providecommand{\tightlist}{
+  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
+
+\begin{document}
+
+\def\milestoneName{ {{- cookiecutter.title -}} }
+\def\milestoneId{ {{- cookiecutter.plan -}} }
+\def\product{Data Management}
+
+\setDocCompact{true}
+
+\title{ {{- cookiecutter.plan }} {{ cookiecutter.title }} Test Plan }
+\setDocRef{\lsstDocType-\lsstDocNum}
+\date{\vcsDate}
+\author{ {{- cookiecutter.author -}} }
+
+\input{history_and_info.tex}
+
+\setDocAbstract{
+This is a stub test report created from a template.
+The content is being prepared.
+}
+
+\maketitle
+
+\section{Introduction}
+Use the GitHub Actions workflow ``docgen from Jira'' to generate this document's content from Jira.
+
+\subsection{References}
+\label{sect:references}
+\renewcommand{\refname}{}
```

Comment:
Why is this here? It is left-over from experimentation, or something that should go in the `lsstdoc` class instead?

Reply:
will check that - its a copy of the other doc in there.. scons was that i neede python 3.10 ok now

Reply:
\renewcommand{\refname}{} is needed in this doc since the bib is in a subseciton if we have to supress the automatic unlabled section